Documentation/KnownIssues/Template/NETFXKnownIssuesTemplate.md

```diff
@@ -0,0 +1,19 @@
+## [|Known Issue Title|]
+// There is no built in way to do comments in Markdown, so this C# style comment used to mark comments. Please REMOVE ALL COMMENTS before submission.
```

Comment:
built in --> built-in
this C# style comment used --> this template uses a C#-style comment